src/redraw.mm

```diff
@@ -360,14 +303,14 @@ - (void) doAction:(RedrawCode::Enum)code withArgc:(int)argc argv:(const object *
             CGSize size = bitmapContextSizeInPoints(self, mCanvasContext);
             NSRect totalRect = {CGPointZero, size};
 
-            totalRect.origin.y += amt * mCharSize.height;
+            totalRect.origin.y = floor(totalRect.origin.y + amt * mCharSize.height);
 
             CGContextSaveGState(mCanvasContext);
             CGContextClipToRect(mCanvasContext, dest);
             drawBitmapContext(mCanvasContext, mCanvasContext, totalRect);
             CGContextRestoreGState(mCanvasContext);
 
-            /* Clear the newly-visible lines */
+            // Clear the newly-visible lines
```

Comment:
Yea, a brief style guide would be nice. There are plenty ones out there. Maybe you should just link to one and write up a "CONTRIBUTE.md"? :)

Just my 2 cents to the comments: If they're one liners I tend to use the "double-slash" style as it doesn't intervene with multi-line comments. Therefore it's easier to comment out big chunks of the code. Maybe you've got other reasons to always use multi-line comments. Anyway, it's a matter of preference and taste, and a styleguide is thus a great idea :+1: